### PR TITLE
Remove internal use method under implementations.

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: sdk/storage/azure-storage-common/src/samples/java/com/azure/storage/common/credentials/SASTokenCredentialJavaDocCodeSnippets.java [hunk 1]
```diff
@@ -38,7 +37,7 @@ public void fromSASTokenString() {
     public void fromQueryParameters() {
         // BEGIN: com.azure.storage.common.credentials.SasTokenCredential.fromQueryParameters#Map
         SasTokenCredential credential = SasTokenCredential
-            .fromQueryParameters(Utility.parseQueryString(url.getQuery()));
+            .fromQueryParameters(StorageImplUtils.parseQueryString(url.getQuery()));
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-common/src/samples/java/com/azure/storage/common/credentials/SASTokenCredentialJavaDocCodeSnippets.java]: Shouldn't use this in our JavaDocs as it is non-public now

Comment (sima-zhu) [reply sdk/storage/azure-storage-common/src/samples/java/com/azure/storage/common/credentials/SASTokenCredentialJavaDocCodeSnippets.java]: Moved them back

Comment (sima-zhu) [reply sdk/storage/azure-storage-common/src/samples/java/com/azure/storage/common/credentials/SASTokenCredentialJavaDocCodeSnippets.java]: parseQueryString and etc function move to StorageImplUtils
JavaDoc code snippet takes a fake query map.

**Pull request merged**